LearnDjango/tests/TestLogIn.py

```diff
@@ -0,0 +1,173 @@
+import unittest
+from .models import MyUser
+

```

Comment:
You'll need to add __from TAInformation.Models.account_type import AccountType__ to use the AccountType enum values.